rules/gcp_audit_rules/gcp_destructive_queries.py

```diff
@@ -0,0 +1,79 @@
+from panther_base_helpers import deep_get
+
+DESTRUCTIVE_STATEMENTS = ("UPDATE", "DELETE", "DROP_TABLE", "ALTER_TABLE", "TRUNCATE_TABLE")
+
+
+def rule(event):
+    return any(
+        [
+            all(
```

Comment:
I would recommend breaking this out into two conditionals. I am having some trouble creating a mental map for how this evaluates.